index.js

```diff
@@ -1,58 +1,108 @@
 // implement your API here
 const express = require("express");
 const server = express();
-const db = require("./data/db.js");
 const port = 8000;
+const db = require("./data/db.js");
+const cors = require("cors");
+
+server.use(express.json());
+server.use(cors());
+
+const sendError = (res, status, msg) => {
+  res.status(status).json({ error: msg });
+  return;
+};
+
+const sendSuccess = (res, status = 200, data, msg) => {
+  res.status(status).json({ data, message: msg });
+  return;
+};
 
 server.get("/", (req, res) => {
   res.send("Hello World, from Node.js!!!!!");
 });
 
 server.post("/api/users", (req, res) => {
-  res.status(201).json({ url: "/api/users", operation: "POST" });
+  const { name, bio } = req.body;
+  if (!name || !bio) {
+    sendError(res, 400, "Provide name and bio");
+    return;
+  }
+  db.insert({ name, bio })
+    .then(response => {
+      sendSuccess(res, 201, response, "User successfully added!");
+    })
+    .catch(() => {
```

Comment:
make sure you pass the param here as that will be the rejected promise that will be caught in the case that there is an error.